### ISSUE TITLE
Videojs requestPictureInPicture() works only once on scrolling to top and bottom

### ISSUE THREAD

**Issue by madefortestonly** [videojs/video.js#6524]
I'm using video.js framework to show a video on page, but there is a problem with PictureInPicture functionality. I'm trying to go in PictureInPicture mode automatically when user scrolls down, and again when user scrolls up (back to the actual video player) exit the PictureInPicture mode.
The first time I scroll down and up it's all working, but any attempt after that fails.
Here's a pen to my code: [Videojs PictureInPicture On Scroll Problem](https://codepen.io/madefortestonly/pen/GRJxMya?editors=1011)

This is the error message I got from the catch block:
`DOMException: Failed to execute 'requestPictureInPicture' on 'HTMLVideoElement': Must be handling a user gesture if there isn't already an element in Picture-in-Picture.
at Html5.requestPictureInPicture (https://vjs.zencdn.net/7.6.6/video.js:22669:23)
at Player.techGet_ (https://vjs.zencdn.net/7.6.6/video.js:26085:34)
at Player.requestPictureInPicture (https://vjs.zencdn.net/7.6.6/video.js:26787:21)
at Player.<anonymous> (pen.js:75:16)
at https://vjs.zencdn.net/7.6.6/video.js:4530:9`

It looks like you need to click on the page for scroll to work for the next attempts because scroll is not part of the user-trusted events.
I tried to simulate a click with Javascript on scrolling up but so far no luck.

Videojs version: v7.6.6

**Comment by gkatsev** [comment]
You cannot enable PiP automatically. It must come from an actual user gesture. You can't emulate it by creating an event.  It isn't possible to do what you're trying to do.
